ansible/group_vars/all/ceos.yml

```diff
@@ -1,4 +1,4 @@
-ceos_image_filename: cEOS64-lab-4.23.2F.tar.xz
-ceos_image_orig: ceosimage:4.23.2F
-ceos_image: ceosimage:4.23.2F-1
+ceos_image_filename: cEOS64-lab-4.25.5.1M.tar
```

Comment:
The new file is not zipped?

Reply:
yes, it's docker-importable 
It's small, just 8mb bigger than cEOS64-lab-4.23.2F.tar.xz
and in our playbook, we download and import directly without unzip, not sure whether previous one is zipped indeed.